modules/sql-engine/src/main/java/org/apache/ignite/internal/sql/engine/util/RexUtils.java

```diff
@@ -197,7 +199,7 @@ public static boolean isIdentity(List<? extends RexNode> projects, RelDataType i
 
     /** Binary comparison operations. */
     private static final Set<SqlKind> BINARY_COMPARISON =
```

Comment:
why we can't use org.apache.calcite.sql.SqlKind#BINARY_COMPARISON ?

Reply:
cause planner generates correlates with IS NOT DISTINCT FROM form, check for example :
AggregatePlannerTest#expandDistinctAggregates